CP1404/prac_06/guitar.py

```diff
@@ -0,0 +1,17 @@
+import datetime
+
+
+class Guitar:
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def get_age(self):
+        return int(datetime.datetime.now().year - self.year)
```

Comment:
Why convert to an int _after_ the calculation? If `year` is not an int, it will crash. If it is an int, the calculation returns an int. It won't be a float.